robottelo/ui/contentviews.py

```diff
@@ -450,6 +462,50 @@ def add_remove_errata_to_filter(self, cv_name, filter_name,
         else:
             self.click(locators['contentviews.remove_errata'])
 
+    def edit_erratum_date_range_filter(
+            self, cv_name, filter_name, errata_types=None, date_type=None,
+            start_date=None, end_date=None, open_filter=True):
+        """Edit Erratum Date Range Filter"""
+        allowed_errata_types = FILTER_ERRATA_TYPE.values()
+        allowed_date_types = FILTER_ERRATA_DATE.values()
```

Comment:
after all the comments applied `allowed_date_types` became single-use variable, presence of which we're trying to avoid.
not a blocker too, and basically it's debatable as this specific single-use variable adds readability.